SCClassLibrary/Common/GUI/Base/QTextView.sc

```diff
@@ -9,35 +9,35 @@ TextView : QAbstractScroll {
 		this.setProperty( \enterInterpretsSelection, bool );
 	}
 
-	editable_ { arg aBool;
-		editable = aBool;
-		this.setProperty( \readOnly, aBool.not );
+	editable_ { arg bool;
+		editable = bool;
+		this.setProperty( \readOnly, bool.not );
 	}
 
-	usesTabToFocusNextView_ { arg aBool;
-		this.setProperty( \tabChangesFocus, aBool );
+	usesTabToFocusNextView_ { arg bool;
+		this.setProperty( \tabChangesFocus, bool );
 	}
 
-	open { arg aString;
-		this.setProperty( \document, aString );
+	open { arg path;
+		this.setProperty( \document, path );
 	}
 
 	string {
 		^this.getProperty( \plainText );
 	}
 
-	string_ { arg aString;
-		this.setProperty( \plainText, aString );
+	string_ { arg string;
+		this.setProperty( \plainText, string );
 	}
 
-	font_ { arg aFont;
-		font = aFont;
-		this.setProperty( \textFont, aFont );
+	font_ { arg argFont;
+		font = argFont;
+		this.setProperty( \textFont, font );
```

Comment:
font is an instance var of View, and after doing font = argFont, 
the two are the same object. 
I find setting to the current state of the instvar clearer than 
referring to the temporary method argument.